server/config/incident.js

```diff
@@ -237,10 +232,10 @@ module.exports = {
   },
 }
 
-function inputsExcludingEmptyStrings(inputs) {
+function inputsExcludingEmptyStrings(attr = 'name', inputs = []) {
   const sanitisedArray = []
```

Comment:
would a filter and util be nicer here, e.g.
const sanitisedArray = inputs.filter(input => !isBlank(input))

Reply:
done but have changed function name to removeEmptyValues